controllers/users.js

```diff
@@ -15,18 +16,30 @@ router.get('/teachersOfSchool', async (req, res, next) => {
 			},
 		});
 
-		const result = users.data.map(user => ({
+		const result = users.data.map((user) => ({
 			_id: user._id,
 			firstName: user.firstName,
 			lastName: user.lastName,
 		}));
 
 		return res.json(result);
-	} catch (e) {
+	} catch (err) {
 		const error = new Error(res.$t('global.text.invalidRequest'));
 		error.status = 400;
 		return next(error);
 	}
 });
 
+router.patch('/language', async (req, res, next) => {
+	try {
+		const { language } = req.body;
+
+		await api(req, { version: 'v3' }).patch('/user/language', { json: { language } });
+		setCookie(res, 'USER_LANG', language);
```

Comment:
mh any idea why we don't need to set the cookie in the nuxt client and it still works?